accepted/rfc-process.md

```diff
@@ -0,0 +1,109 @@
+    *   This list includes the GitHub handle for each individual stakeholder, organized into stakeholder groups. 
+    *   The individual who filed the motion to finalize is automatically checked off.
+*   Once _any_ stakeholder from a _different_ group has signed off, the RFC will move into a 10 day **final comment period** (FCP), long enough to ensure that other stakeholders have at least a full business week to respond.
+*   During FCP, any stakeholder can raise an **objection** using a syntax understood by the bot. Doing so aborts FCP and labels the RFC as `blocked-by-stakeholder` until the objection is formally resolved (again using a special comment syntax).
+*   Finally, the RFC is automatically merged/close if either:
+    *   The FCP elapses without any objections.
+    *   A stakeholder from _each_ group has signed off, short-cutting the waiting period.
+
+## What goes into an RFC?
+
+RFCs will follow a format inspired by Rust, but significantly simplified. RFCs are markdown files containing the following sections (which will be laid out in a template file):
+
+*   **Summary**. A ~one paragraph overview of the RFC.
+*   **Motivation**. What problem does the RFC solve?
+*   **Proposal**. The meat of the RFC.
+*   **Rationale and alternatives**. A discussion of tradeoffs: why was the proposal chosen, rather than alternatives?
+*   **Open questions**. Often an RFC is initially created with a broad proposal but some gaps that need community input to fill in.
+
+### Draft RFCs
+
+It is encouraged to use the RFC process to discuss ideas early in the design phase, before a _full_ proposal is ready. Such RFCs should be marked as [a draft PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests), and contain the following sections:
+
+*   **Motivation**. What problem are you ultimately hoping to solve?
+*   **Proposal sketch**. At least a short sketch of a possible approach. Beginning discussion without _any_ proposal tends to be unproductive.
+*   **Open questions**. This section is especially important for draft RFCs: it’s where you highlight the key issues you are hoping that discussion will address.
+
+RFCs cannot be merged in draft form. Before any motion to merge, the draft should be revised to include all required RFC sections, and the PR should be changed to a standard GitHub PR.
+
+## What requires an RFC?
+
+When should you open an RFC, rather than just writing code and opening a traditional PR?
+
+*   When the work involves changes that will significantly affect stakeholders or project contributors. Each project may provide more specific guidance. Examples include:
+    *   Major architectural changes
+    *   Major new features
+    *   Simple changes that have significant downstream impact
+    *   Changes that could affect guarantees or level of support, e.g. removing support for a target platform
```

Comment:
Or adding support for a platform with the expectation that it must be maintained going forward.